kvrocks.conf

```
@@ -462,6 +462,16 @@ profiling-sample-record-threshold-ms 100
 # 0-7am every day.
 compaction-checker-range 0-7
 
+# When the compaction checker is triggered, the db will periodically pick the SST file
+# with the highest "deleted percentage" (i.e. the percentage of deleted keys in the SST 
+# file) to compact, in order to free disk space. 
+# However, if a specific SST file was created before "force-compact-file-age" seconds ago, 
+# and its percentage of deleted keys is higher than 
+# "force-compact-file-min-deleted-percentage", it will be forcely compacted as well.
+
+# force-compact-file-age 172800
+# force-compact-file-min-deleted-percentage 10
```

```suggestion
force-compact-file-age 172800
force-compact-file-min-deleted-percentage 10
```

Add defalut value description，min & max value.

---

Done.